manifests/setup.pp

```diff
@@ -37,6 +37,8 @@
 
   $script_owner = $::osfamily ? { 'windows' => undef, default => $::id }
 
+  $script_group = $script_owner ? { 'root' => 'root', default => undef }
+
```

Comment:
Darn, you're certainly right. Do you have any idea how to solve my problem, then? It's annoying to have puppet default to the remote file owner+group instead of the local user and his default group...